### PR TITLE
Update AdRoll.xml

### PR DESCRIPTION
**Pull request by pipboy96**

### CODE REVIEW

File: src/chrome/content/rules/AdRoll.xml [hunk 1]
```diff
@@ -1,106 +1,26 @@
...
-
-	<rule from="^http://feedback\.adroll\.com/"
-		to="https://userecho.com/" />
-
-	<rule from="^http:"
-		to="https:" />
+	<securecookie host=".+" name=".+" />
 
+	<rule from="^http://adroll\.com/" to="https://www.adroll.com/" />
+	<rule from="^http://support\.adroll\.com/" to="https://help.adroll.com//" />
```
Comment (J0WI) [inline src/chrome/content/rules/AdRoll.xml]: `//` seems to be a typo

Comment (pipboy96) [reply src/chrome/content/rules/AdRoll.xml]: @J0WI It is not. I copied the exact URL `http://support.adroll.com/` redirects to:
```
$ curl -I http://support.adroll.com/
HTTP/1.1 302 Moved Temporarily
Server: nginx/1.12.2
Date: Sun, 10 Feb 2019 17:17:29 GMT
Content-Type: text/html
Content-Length: 161
Connection: keep-alive
Location: https://help.adroll.com//

curl -I http://support.adroll.com/cf2xbfnlmumhsasy6tvya26fne
HTTP/1.1 302 Moved Temporarily
Server: nginx/1.12.2
Date: Sun, 10 Feb 2019 17:19:37 GMT
Content-Type: text/html
Content-Length: 161
Connection: keep-alive
Location: https://help.adroll.com//cf2xbfnlmumhsasy6tvya26fne
```

Comment (J0WI) [reply src/chrome/content/rules/AdRoll.xml]: Let's remove the second `/ `, the server will handle it.

Comment (pipboy96) [reply src/chrome/content/rules/AdRoll.xml]: @J0WI Done.

**Pull request merged**